science/mutationpp/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+name                    mutationpp
+github.setup            mutationpp Mutationpp 1.0.0 v
+revision                0
+categories              science
+license                 LGPL-3+
+maintainers             {@rubendibattista} openmaintainer
+platforms               darwin
+description             The MUlticomponent Thermodynamic And Transport library for IONized gases in C++
+long_description        Mutation++ is an open-source library originally \
+                        developed at the von Karman Institute for Fluid Dynamics, designed to couple \
+                        with conventional computational fluid dynamics codes to provide thermodynamic, \
+                        transport, chemistry, and energy transfer properties associated with subsonic \
+                        to hypersonic flows.
+
+homepage                https://github.com/mutationpp/Mutationpp
```

Comment:
This `homepage` line is redundant and should be removed because it is already set to this by the `github.setup` line above.